testing/test_skipping.py

```diff
@@ -350,7 +351,7 @@ def test_raises():
         ])
 
 
-class TestXFailwithSetupTeardown:
+class TestXFailwithSetupTeardown(object):
```

Comment:
No need to subclass from `object` here... all other test classes in pytest's test suite don't subclass, so I feel we should be consistent